docs/reference/quickstart/getting-started.asciidoc

```diff
@@ -1,83 +1,145 @@
+You'll learn how to create an index, add data as documents, work with dynamic and explicit mappings, and perform your first basic searches.
 
-[discrete]
-[[add-data]]
-=== Add data
-
-You add data to {es} as JSON objects called documents.
-{es} stores these
-documents in searchable indices.
+[TIP]
+====
+The code examples in this tutorial are in {kibana-ref}/console-kibana.html[Console] syntax by default.
+You can {kibana-ref}/console-kibana.html#import-export-console-requests[convert into other programming languages] in the Console UI.
+====
 
 [discrete]
-[[add-single-document]]
-==== Add a single document
+[[getting-started-prerequisites]]
+=== Prerequisites
 
-Submit the following indexing request to add a single document to the
-`books` index.
-The request automatically creates the index.
+Before you begin, you need to have a running {es} cluster.
+The fastest way to get started is with a <<run-elasticsearch-locally,local development environment>>.
+Refer to <<elasticsearch-intro-deploy,Run {es}>> for other deployment options.
 
 ////
 [source,console]
 ----
 PUT books
+PUT my-explicit-mappings-books
 ----
 // TESTSETUP
 
 [source,console]
 --------------------------------------------------
 DELETE books
+DELETE my-explicit-mappings-books
 --------------------------------------------------
 // TEARDOWN
 
 ////
 
+[discrete]
+[[getting-started-index-creation]]
+=== Step 1: Create an index
+
+Create a new index named `books`:
+
+[source,console]
+----
+PUT /books
+----
+// TEST[skip: index already setup]
+
+A `200` response indicates the index was created successfully.
+
+.Expand to see example response
+[%collapsible]
+===============
+[source,console-result]
+----
+{
+  "acknowledged": true,
+  "shards_acknowledged": true,
+  "index": "books"
+}
+----
+// TEST[skip: index already setup]
+===============
+
+[discrete]
+[[getting-started-add-documents]]
+=== Step 2: Add data to your index
+
+[TIP]
+====
+This tutorial uses {es} APIs, but there are many other ways to
+<<es-ingestion-overview,add data to {es}>>.
+====
+
+You add data to {es} as JSON objects called documents.
+{es} stores these
+documents in searchable indices.
+
+[discrete]
+[[getting-started-add-single-document]]
+==== Add a single document
+
+Submit the following indexing request to add a single document to the
+`books` index.
+
+[TIP]
+====
+If the index didn't already exist, this request would automatically create it.
+====
+
 [source,console]
 ----
 POST books/_doc
 {"name": "Snow Crash", "author": "Neal Stephenson", "release_date": "1992-06-01", "page_count": 470}
 ----
-// TEST[s/_doc/_doc?refresh=wait_for/]
+// TEST[continued]
 
-The response includes metadata that {es} generates for the document including a unique `_id` for the document within the index.
+The response includes metadata that {es} generates for the document, including a unique `_id` for the document within the index.
 
-.Expand to see example response
+.Expand to see example response with annotations
```

Comment:
the "w/ annotations" I think is unnecessary here

Reply:
💯